src/patternfly/components/Chips/examples/chip-complex-example.hbs

```diff
@@ -0,0 +1,135 @@
+{{#> chip-group chip-group--attribute='role="group of chips"'}}
+  {{#> chip}}
+    {{#> chip-text}}
+      Value One
+    {{/chip-text}}
+    {{#> button button--HasTimesCircleIcon="true" button--attribute='aria-label="Remove"'}}
+    {{/button}}
+  {{/chip}}
+  {{#> chip}}
+    {{#> chip-text}}
+      Value Two
+    {{/chip-text}}
+    {{#> button button--HasTimesCircleIcon="true" button--attribute='aria-label="Remove"'}}
+    {{/button}}
+  {{/chip}}
+  {{#> chip}}
+    {{#> chip-text}}
+      Value Three
+    {{/chip-text}}
+    {{#> button button--HasTimesCircleIcon="true" button--attribute='aria-label="Remove"'}}
+    {{/button}}
+  {{/chip}}
+  {{#> chip}}
+    {{#> chip-text}}
+      Value Four
+    {{/chip-text}}
+    {{#> button button--HasTimesCircleIcon="true" button--attribute='aria-label="Remove"'}}
+    {{/button}}
+  {{/chip}}
+  {{#> chip chip--modifier="pf-m-overflow"}}
+  {{#> button}}
+    {{#> chip-text}}
+      4 more
+    {{/chip-text}}
+  {{/button}}
+{{/chip}}
+{{/chip-group}}
+{{#> chip chip--modifier="pf-m-overflow"}}
+  {{#> button}}
```

Comment:
the button should always be used with a modifier class. these buttons might be so unique they could just be their own element, and not use `.pf-c-button`?